### DISCUSSION TITLE
如何在 Notion 使用简悦的阅读模式

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2106] [站点适配]
> 用过 Notion 用户都知道 **Notion 自带的导出 Markdown 功能会导致图片丢失**，尤其是通过 Notion 分享出去的页面均存在此问题，所以我使用简悦适配了 **Notion 的阅读模式**。

接入简悦的阅读模式后，就可以使用导出、标注、加入稍后等操作。

https://user-images.githubusercontent.com/81074/159113564-6fe7bd1b-2a75-4047-b8e5-49b30bd27472.mp4

# 如何导入

进入 [站点集市](https://simpread.pro/sites)，一共有两个关于 Notion 的适配规则：

- Notion 分享页 [Notion.site](https://simpread.ksria.cn/sites/details/PYK6c7l880)

- Notion 编辑页 [Notion.so](https://simpread.ksria.cn/sites/details/JPbELFpvSS)

# 如何配置

因为 Notion 属于动态加载页面的加载方式，所以需要将这两个站的域名加入到 [延迟加载列表](http://ksria.com/simpread/docs/#/%E8%AF%8D%E6%B3%95%E5%88%86%E6%9E%90%E5%BC%95%E6%93%8E?id=%e5%bb%b6%e8%bf%9f%e5%8a%a0%e8%bd%bd)

位置：选项页 → 高级设定 → 词法分析引擎，将 `https://*.notion.site/*/*` 与 `https://www.notion.so/*` 加入即可，如下图：

![image](https://user-images.githubusercontent.com/81074/135816830-f6c5cb4a-c9cd-4771-b2b9-776daf3e17c9.png)

# 适配的 Block

- [x] header → 标题 
- [x] sub_sub_header → 标题
- [x] text → 文本段落
- [x] bulleted_list → 无序列表
- [x] numbered_list → 有序列表
- [x] image → 图片
- [x] bookmark → 书签
- [x] Table View → 表格
- [x] code → 代码段
- [x] video → 视频

# 未适配的 Block

大部分跟非文章展示类的 Block 均没有适配，如：Toggle Block · Table of contents 等。

# 效果

进入 [示例一](https://www.notion.so/1505c8ae200e4e5fb0accdae24839190) 或 [示例二](https://www.notion.so/P-A-R-A-Notion-19909e5aac3049d887197dcfb1e97fd5) 等页面加载完毕后使用 **快捷键** 或 **点击右上角 icon** 即可进入，导出效果看这里 [长截图](https://user-images.githubusercontent.com/81074/117267149-8e3bc880-ae88-11eb-91dc-8a9f14dca0c3.jpeg)。

# 引申

关于简悦与 Notion 更多用法，[请看汇总](https://github.com/Kenshin/simpread/discussions/2586)。

**Comment by ghost** [comment]
我使用中发现一个问题 
![正文中较大的图标](https://user-images.githubusercontent.com/482437/117277700-add7ee80-ae92-11eb-8721-d2fe8af3db66.png)

**Reply by Kenshin** [reply]
应该不会存在这个问题，我看了你的 URL 应该是示例给出的，能否稳定复现？

**Reply by ghost** [reply]
> 应该不会存在这个问题，我看了你的 URL 应该是示例给出的，能否稳定复现？

抱歉，对照了一下设置，应该是我开启纯粹模式的缘故。

**Reply by Kenshin** [reply]
> 抱歉，对照了一下设置，应该是我开启纯粹模式的缘故。

okay 没问题就好~

**Comment by Joel2561** [comment]
绝了！！以前只知道简悦可以适配notion页面但一直没试过，今天一用才知道居然这么牛逼！！！另外少楠老师的这篇文章写的也非常棒，学习了！
另外一开始我也开启了纯粹模式导致出现和楼上同样的情况，可以考虑在原文里提一下。

**Reply by Kenshin** [reply]
哈哈，后知后觉了。（好东西不怕晚~

> 另外一开始我也开启了纯粹模式导致出现和楼上同样的情况，可以考虑在原文里提一下。

okay 谢谢提醒，已加。

**Comment by Karllistark** [comment]
你好，大佬，我遇到一个问题，就是notion阅读模式会出现一个notion的 slogan，如图，原始页面是没有的，这个有解决办法嘛？还是我设置问题？
![difference](https://github.com/Kenshin/simpread/assets/140293800/9de87978-85d6-48b0-8503-1834dccbe006)

**Reply by Kenshin** [reply]
嗨，这是阅读模式自带的「描述」段落的内容，你可以用 https://github.com/Kenshin/simpread/discussions/2620 这个插件来删除描述段落。

**Reply by Karllistark** [reply]
谢谢 解决了:D

**Reply by Kenshin** [reply]
okay 解决就好~

**Comment by Jack980506** [comment]
怎么我的还是没成功？已经把noton的网址和第三方适配元都加进去了，在notion里读文章进入阅读模式后，只显示开头部分内容，没法下拉显示剩余内容。
![image](https://github.com/user-attachments/assets/ee06cc31-a09e-484d-b98a-9bb93a2172cb)

**Reply by Kenshin** [reply]
滚动条无法显示是另外的问题，你需要使用 https://simpread.ksria.cn/plugins/details/E2CUZ2agRP 这个插件。

**Reply by Jack980506** [reply]
试了下，安装了您推的这个插件也无法往下滑动，但是AI总结是可以总结全文内容

**Reply by Kenshin** [reply]
是否使用了同步助手的自动同步功能？如果是的话，请使用 https://www.yuque.com/kenshin/simpread/ugu11ptr2sx9zd0k 安装
如果没有的话，把截图的 URL 给我看下。

**Reply by Jack980506** [reply]
按照文档操作试了下依然不行，notion页面的链接如下：https://www.pmthinking.com/6ec4d577a9c14da6b688f19343cc17fd

**Reply by Kenshin** [reply]
@Jack980506 

即便没有安装这些规则，也可以直接进入

<img width="399" alt="image" src="https://github.com/user-attachments/assets/cab2a7a1-cc80-481b-8f27-b8dc7c889e51">

<img width="1795" alt="image" src="https://github.com/user-attachments/assets/508f25c2-5be9-4d00-b470-157af3944f5d">

**Reply by Jack980506** [reply]
哥，是可以进入阅读模式，但是进入后无法往下滑动，只能看开头内容

**Reply by Kenshin** [reply]
试试安装 https://github.com/Kenshin/simpread/discussions/6756
